tests/ci/docker_server.py

```diff
@@ -186,8 +186,12 @@ def gen_tags(version: ClickHouseVersion, release_type: str) -> List[str]:
     return tags
 
 
-def buildx_args(bucket_prefix: str, arch: str) -> List[str]:
-    args = [f"--platform=linux/{arch}", f"--label=build-url={GITHUB_RUN_URL}"]
+def buildx_args(bucket_prefix: str, arch: str, version: ClickHouseVersion) -> List[str]:
+    args = [
+        f"--platform=linux/{arch}",
+        f"--label=build-url={GITHUB_RUN_URL}",
+        f"--label=com.clickhouse.ClickHouseVersion.githash={version.githash}",
```

Comment:
It's better to use `git.sha` here. `type=version_arg` returns `ClickHouseVersion` w/o `_git`.